paragon/css/core/variables.css

```diff
@@ -63,9 +63,6 @@
   --pgn-size-popover-border-width: 0px;
   --pgn-size-nav-pills-border-radius: 0;
   --pgn-size-image-thumbnail-border-radius: 0;
-  --pgn-size-form-input-radius-border-sm: 0;
```

Comment:
These have been removed as the default border radius found in Paragon matches the Elm theme Figma